libgdf/src/gdf_table.cuh

```diff
@@ -708,146 +590,35 @@ public:
     hash_value_type hash_value{0};
 
     // If num_columns_to_hash is zero, hash all columns
-    if(0 == num_columns_to_hash)
+    if(0 == num_columns_to_hash) 
     {
       num_columns_to_hash = this->num_columns;
     }
 
+    // Generic lambda to hash an element in a column and combine 
+    // it with the previous hash value
+    auto hash_element = [&hash_value](auto dispatched_type_var, size_type row_index, 
```

Comment:
So why not call this `auto hash_combine_with_element = [&previous_elements_combined_hash](etc. etc.` ?